ci/displayCodeCoverage.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+#
+# Show Code coverage NodeJS for Linux
+#
+# - TARGET_DOCKER_TEST_IMAGE - the target Docker image key. It must be registered in _init.sh
+#
+set -o pipefail
+THIS_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+echo $THIS_DIR
+echo $(ls)
+npx nyc report --reporter=text
```

Comment:
maybe we can get rid of this script?

Reply:
Yes It is possibly to simply run a npx command